wave_kinematics/include/wave/kinematics/two_wheel.hpp

```diff
@@ -0,0 +1,40 @@
+/** @file
+ * @ingroup kinematics
+ */
+
+#ifndef WAVE_KINEMATICS_TWOWHEEL_HPP
+#define WAVE_KINEMATICS_TWOWHEEL_HPP
+
+#include <float.h>
+#include <iostream>
+
+#include "wave/utils/utils.hpp"
+
+namespace wave {
+/** @addtogroup kinematics
+ *  @{ */
+
+/** Generic two wheel robot motion model */
```

Comment:
Both are different models though? (looking at slides ME597 - 3 Motion Modelling; Two wheel slide 34, Bicycle slide 46), plus the bicycle model takes in a slightly different inputs.